README.md

```
@@ -47,6 +48,10 @@ Create cable connection in NetBox from LibreNMS links data.
 ### IP Address Sync
 Create IP address in NetBox from LibreNMS device IP data.
 
+### VLAN Sync
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)